test/manipulation.test.js

```diff
@@ -1338,6 +1338,25 @@ describe('manipulation', function() {
         });
       }
     });
+    describe('replaceById', function() {
```

Comment:
I thought I did. the describe block is within the else block, so the previous `bdd.describeif` is applicable here. I will re-arrange the code one more time and re-commit